packages/core-mobile/app/screens/bridge/Bridge.tsx

```diff
@@ -320,8 +323,8 @@ const Bridge: FC = () => {
 
       if (error || !hash) {
         // do not show the error when the user denied the transfer
-        if (error === 'User declined the transaction') {
-          Logger.error(error)
+        if (isUserRejectedError(error)) {
```

Comment:
the error string is different now since we are using rpc errors. updated to use the same logic in other places to check for rejected error.